syncopy/io/utils.py

```diff
@@ -77,11 +77,8 @@ def cleanup(older_than=24, interactive=True, **kwargs):
     """
 
     # Make sure age-cutoff is valid
-    try:
-        scalar_parser(older_than, varname="older_than", ntype="int_like",
+    scalar_parser(older_than, varname="older_than", ntype="int_like",
```

Comment:
yes... that's how these "parsers" should be used, they have their own exception handling/messagin